src/refspec.c

```diff
@@ -160,6 +162,27 @@ void git_refspec__free(git_refspec *refspec)
 	memset(refspec, 0x0, sizeof(git_refspec));
 }
 
+int git_refspec_parse(git_refspec **refspec, const char *input, int is_fetch) {
+	git_refspec *new_refspec;
+	assert(refspec && input);
+
+	new_refspec = git__malloc(sizeof(git_refspec));
+	GITERR_CHECK_ALLOC(new_refspec);
+
+	if (!git_refspec__parse(new_refspec, input, is_fetch)) {
+		git__free(new_refspec);
+		return -1;
+	}
+
+	*refspec = new_refspec;
+	return 0;
+}
+
+void git_refspec_free(git_refspec *refspec) {
```

Comment:
Same with this `{`.

Reply:
Sorry, didn't noticed that. Should be fixed.